assets/dev/js/editor/regions/panel/component.js

```diff
@@ -24,8 +24,12 @@ export default class Component extends ComponentBase {
 
 	defaultCommands() {
 		return {
-			open: () => elementor.getPanelView().modeSwitcher.currentView.setMode( 'edit' ),
-			close: () => elementor.getPanelView().modeSwitcher.currentView.setMode( 'preview' ),
+			open: () => new class Open extends $e.modules.CommandBase {
```

Comment:
A class as a command??

Reply:
Most of the commands are like that. ( non using class is optional ) take look at `document/elements` for exmaple `wp-content/plugins/elementor/assets/dev/js/editor/document/elements/commands`.

Reply:
You should export it to a new file